zipline/finance/performance/period.py

```diff
@@ -212,6 +241,24 @@ def rollover(self):
             else:
                 del self._payout_last_sale_prices[asset]
 
+    def subdivide_period(self, capital_change):
+        self.calculate_performance()
+
+        # Apply the capital change to the ending cash
+        self.ending_cash += capital_change
+
+        # Increment the total capital change occurred within the period
+        self._total_intraperiod_capital_change += capital_change
+
+        # Divide the period into subperiods
+        self.subperiod_divider = SubPeriodDivider(
```

Comment:
Looks good to me. I think the divider object worked out well with making the before and after of the capital change clearer.